server.go

```diff
@@ -174,6 +218,11 @@ func main() {
 
 	campaignGroup.PUT(fmt.Sprintf("%s/:%s", ADD, PARAM_CAMPAIGN_NAME), addCampaign)
 
+	// Scoreing related endpoints and group
+	scoreGroup := e.Group(SCORE_EVENT)
+
+	scoreGroup.POST("/new", newScore)
```

Comment:
You could technically add `"/new"` to your list of endpoint stuff further up

Reply:
done c51c7f2